p12.ts

```diff
@@ -0,0 +1,18 @@
+declare module "stats" {
```

Comment:
The solution is correct although the rest of the code is missing in order to run it